packages/services/f-analytics/src/services/_tests/analytics.service.test.js

```diff
@@ -348,20 +348,20 @@ describe('Analytic Service ::', () => {
             }
         );
 
-        it('should set conversationId and requestId values when provided', () => {
+        it('should set requestId value when provided', () => {
             // Arrange
             const expected = {
                 ...defaultState.pageData,
                 group: 'test-feature-name',
                 name: 'test-page-name',
-                conversationId: '460cc3a8-83f7-4e80-bb46-c8a69967f249',
+                conversationId: undefined,
```

Comment:
I think you can remove this line as the `defaultState.pageData` sets the convo id to undefined 👍

Reply:
I did think this too, but I thought the fact it was explicitly undefined might make the tests a bit easier to understand.

Reply:
But the test is only focusing on the `requestid` and has little to do with the convo id at this point.